ansible/roles/data-products-deploy/templates/model-config.json.j2

```diff
@@ -2286,6 +2286,27 @@
     "appName": "Assessment Dashboard Metrics",
     "deviceMapping": false
   },
+  "admin-user-reports":{
+    "search": {
+      "type": "none"
+    },
+    "model": "org.ekstep.analytics.job.StateAdminReportJob",
+    "modelParams": {
+      "sparkCassandraConnectionHost": "'$sunbirdPlatformCassandraHost'",
+      "sparkElasticsearchConnectionHost": "'$sunbirdPlatformElasticsearchHost'"
```

Comment:
values should J2 variables. Please check other dataproduct config for the same variables.

Reply:
ok, done.